js/client-chat.js

```diff
@@ -808,7 +808,25 @@
 					}
 				}), 'text');
 				return false;
-
+				
+			case 'regdate':
+				if (!target) target = app.user.get('userid');
+				target = toId(target);
+				var self = this;
+				$.get("http://pokemonshowdown.com/users/" + target + ".json",
```

Comment:
@xfix, I mean I don't know if `http://play.pokemonshowdown.com/` to `http://pokemonshowdown.com/` is cross-origin.

Another thought: `*.psim.us` to `pokemonshowdown.com` definitely is.